content/en/docs/kubernetes/collector/components.md

```diff
@@ -22,6 +22,8 @@ Components covered in this page:
   cluster-level metrics and entity events.
 - [Kubernetes Objects Receiver](#kubernetes-objects-receiver): collects objects,
   such as events, from the Kubernetes API server.
+- [Prometheus Receiver](#kubernetes-objects-receiver): receives metrics in
```

Comment:
Copy-paste error:
```suggestion
- [Prometheus Receiver](#prometheus-receiver): receives metrics in
```

In know that we wouldn't get the same effect, but the page has an autogenerated TOC:

> <img width="1655" alt="Screen Shot 2023-07-31 at 17 04 35" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/5ae9397d-2298-437e-b3fa-b28e8f8875ca">

Might it be worth dropping this autogenerated one?

Actually, if ever this page is split into separate pages, then we'll get the equivalent of the manually added TOC above automatically for the landing page -- i.e., including the description.